common/php/class-module.php

```diff
@@ -219,7 +219,7 @@ function get_post_status_friendly_name( $status ) {
 		} else if ( array_key_exists( $status, $builtin_stati ) ) {
 			$status_friendly_name = $builtin_stati[$status];
 		}
-		return $status_friendly_name;
+		return esc_html__( $status_friendly_name, 'edit-flow');
```

Comment:
Good job adding the internationalization. Can you elaborate on why we'd want to escape here? It seems that we would be early escaping instead of escaping on output. There's a bit more information here: https://vip.wordpress.com/documentation/validating-sanitizing-escaping/#always-escape-late :)